### ISSUE TITLE
404 error when scanning

### ISSUE THREAD

**Issue by mxmehl** [scanoss/sbom-workbench#730]
When running the workbench on projects, I run into 404 errors such as these:

```
Scan Paused
Error: Scanning
Reason:
HTTP Status Code: 404
Server Response:
404 page not found

Debug file located at ...
```

![image](https://github.com/scanoss/sbom-workbench/assets/6170081/32e165ac-570c-46d2-8871-397c76f0e1da)

This occured when importing [github-org-manager](https://github.com/OpenRailAssociation/github-org-manager/), but it happens with any other project I tested as well.

The mentioned log file is here: [bad_request-1714661456073-3d07b723-9685-4c4d-9672-1c6f239eb1c6.txt](https://github.com/scanoss/sbom-workbench/files/15190128/bad_request-1714661456073-3d07b723-9685-4c4d-9672-1c6f239eb1c6.txt)

The complete content of the directory with files such as `filter.json` or `project.log` can be found here: [github-org-manager.zip](https://github.com/scanoss/sbom-workbench/files/15190171/github-org-manager.zip)

I installed the latest version of the application on my Windows work machine as downloaded from the scanoss website.

Any idea what's going wrong? Thanks already for your insights!

**Comment by francostramana** [comment]
Dear @mxmehl,

Thank you for reporting this issue and for providing us with all the necessary logs, which are crucial for resolving any inconvenience.

It appears that the connection data you're using for scanning is incorrectly configured, and for security reasons, it's not included in the logs.

If you have a dedicated account, please ensure that the correct URL is **https://api.scanoss.com.** A few weeks ago, the old URL (https://scanoss.com/api) was deprecated entirely.

You can update this configuration from the UI panel under the "Settings" option.

Please confirm if this is the issue. Otherwise, we will continue investigating.

**Comment by mxmehl** [comment]
Thank you for the quick response. I've switched the URL to `https://api.scanoss.com/` but now I'm getting 401 (Unauthorized) errors.

I'm not having any subscription. Is there another API URL for free users, or did you shut this off?

**Comment by francostramana** [comment]
Thank you for reaching out again. Indeed, the URL you provided earlier is for dedicated accounts.

However, the open and free OSSKB service is still operational and is the default URL used by SBOM Workbench.

Nevertheless, let's try configuring it manually to see if that's the issue.

API_URL: https://api.osskb.org/ (without API key)

**Comment by francostramana** [comment]
Dear @mxmehl,

Indeed, the version you're using did not have the OSSKB endpoints updated. Nevertheless, you can either configure them manually or upgrade to the new available version (v1.12.4), which already includes these updates.

I've closed this issue, but feel free to open a new one if you need further assistance or have any other queries.

Thank you for spotting this.

Regards
Franco
